### PR TITLE
Removing hardcoded access token

### LINKED ISSUE #291: Remove hardcoded access token
The hardcoded access token should be [removed](https://github.com/mapbox/mapbox-android-demo/blob/master/MapboxAndroidDemo/src/main/res/values/strings.xml#L9) and rotated and use the new Gradle script to integrate environment values.

cc: @cammace @langsmith @Guardiola31337

### PR DESCRIPTION
**Pull request by langsmith**
Resolves #291 . Based on @cammace 's previous work in #294

### CODE REVIEW

**Comment by langsmith** [comment]
@cammace , please take a look when you'd like. No rush on my end.

File: SharedCode/gradle-config.gradle [hunk 1]
```diff
@@ -29,8 +29,15 @@ task secretKeysSetup {
...
 
+        String mapboxAccessToken = "$System.env.MAPBOX_ACCESS_TOKEN"
+        if (mapboxAccessToken == "null") {
+            System.out.println("You should set the MAPBOX_ACCESS_TOKEN environment variable.")
+            mapboxAccessToken = "YOUR_MAPBOX_ACCESS_TOKEN_GOES_HERE"
+        }
+
         String tokenFileContents = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n" +
                 "<resources>\n" +
+                "    <string name=\"access_token\">" + mapboxAccessToken + "</string>\n" +
```
Comment (cammace) [inline SharedCode/gradle-config.gradle]: translatable should be set to false right?

**Comment by langsmith** [comment]
>Shouldn't we add the developer-config.xml file to the .gitignore file too?

Was already added to the repo before this pr  https://github.com/mapbox/mapbox-android-demo/blob/master/.gitignore#L9-L10

File: SharedCode/gradle-config.gradle (hunk 1 above)
Comment (langsmith) [reply SharedCode/gradle-config.gradle]: Correct. Fixed.

**Pull request merged**